src/components/Comment/Comment.tsx

```diff
@@ -408,6 +413,7 @@ export const Comment = ({
 													regularFont,
 													svgOverrides,
 													cssReplyBetaDisplayName,
+													linkOverrides,
```

Comment:
Is this version of Source introducing a new problem that we need to now fix?

Reply:
lots of the override styles seem not to work anymore with updated source. I think it might be worth taking a step back and trying to find a more stable work around for things we do not feel are right

Reply:
Let's chat! 🗣️